### PR TITLE
Clarify our expectations on hotel dates

### PR DESCRIPTION
**Pull request by peterjc**
Perhaps this can be phrased more elegantly/concisely/clearly?

Exceptional circumstances might include international flight schedules which are not daily, or a layover on some long distance routes. I don't think we need to give examples here though.

### CODE REVIEW

File: Travel_fellowships.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ The OBF Event Fellowships are available for the awardees to cover conference reg
 
 The fellowship can be used to cover conference registration fees and direct travel costs (such as airfare, train, metro, bus and taxi), hotel, poster printing (if presenting a poster at the conference), and/or child-care up to a value of USD 1,000. 
 Note that rental car expenses and personal car mileage are not eligible for reimbursement. 
+Hotel accommodation earlier than the night before the first day of the event, or beyond the night the event ends, requires exceptional justification.
```
Comment (hlapp) [inline Travel_fellowships.md:76]: I guess what _exceptional_ means in practice is in the eye of the beholder, i.e., ambiguous? Also, didn't we want to add that approval is at the Board's discretion? Stating both seems at least partly redundant, so I would perhaps simplify to the following:

Comment (hlapp) [reply Travel_fellowships.md:76]:
```suggestion
Lodging for earlier than the night before the first day of the event, or beyond the night the event ends, requires justification and is subject to Board approval.
```

Comment (hlapp) [reply Travel_fellowships.md:76]: BTW I just notice that we use explicitly _hotel_ above in the enumeration in line 74 (2 lines before this change). Perhaps worth generalizing this to _accommodation_ or _lodging_?

Comment (peterjc) [reply Travel_fellowships.md]: Yes, hotels is a little too narrow isn't it.

**Comment by nlharris** [comment]
This looks good to me now

**Comment by nlharris** [comment]
two more reviews needed to merge. @hzahroh @kipkurui 
Please consider changing your GH notification settings to make sure you see notifications about issues and PRs in this repo!

**Review by nlharris — APPROVED** [review]

**Review by kipkurui — APPROVED** [review]

**Review by gedankenstuecke — APPROVED** [review]

**Review by hzahroh — APPROVED** [review]

**Pull request merged**